Automattic/wp-calypso
# Plans: Highlight popular plan

Highlights popular plan as designed in #6761 

## Visuals

![zrzut ekranu 2016-07-14 o 19 44 30](https://cloud.githubusercontent.com/assets/3775068/16849600/7ac97122-49fb-11e6-8e26-44ccbec6d3b2.png)
![zrzut ekranu 2016-07-14 o 19 44 44](https://cloud.githubusercontent.com/assets/3775068/16849601/7d3ccbac-49fb-11e6-8ba9-50f6227d10ae.png)
![zrzut ekranu 2016-07-14 o 19 44 56](https://cloud.githubusercontent.com/assets/3775068/16849606/7f5364d2-49fb-11e6-934a-308123e5053c.png)


## Testing

1. Build with `ENABLE_FEATURES=manage/plan-features make run`
2. Go to http://calypso.localhost:3000/plans/features/$site
3. See Premium beautifully highlighted, dwarfing every level of beauty you have witnessed.
4. Profit

CC @rralian @gwwar @lamosty @apeatling 

Test live: https://calypso.live/?branch=update/popular-plan
